### ISSUE TITLE
TransE missing normalisation? 

### ISSUE THREAD

**Issue by darvid7** [thunlp/TensorFlow-TransX#21]
Hi! Sorry if this is a noob question.
In the paper, the authors mention 'at each main iteration of the algorithm, the embedding vectors of the entities are first normalized'. Is this missing from `TransE` because I can't seem to find the code that does it?
Thanks!

**Comment by THUCSTHanxu13** [comment]
From our experiment results on WN18 and FB15K, non-normalized models can perform as well as normalized models and be trained more efficient. You can try to compare these models on other data sets.

For adding normalization, just using tf.nn.l2_normalize() is ok like [TransE](https://github.com/thunlp/OpenKE/blob/OpenKE-Tensorflow1.0/models/TransE.py)
